src/uaserver.cpp

```diff
@@ -60,6 +103,11 @@ void UaServer::start()
         OPEN62541_COMPAT_LOG_AND_THROW(std::runtime_error, "UA_Server_new failed");
     UA_ServerConfig* config = UA_Server_getConfig(m_server);
     UA_ServerConfig_setMinimal(config, m_endpointPortNumber, nullptr);
+    config->logger = logItLogger;
+    Log::LogComponentHandle handle = Log::getComponentHandle("open62541");
+    if (handle == Log::INVALID_HANDLE)
+    	handle =  Log::registerLoggingComponent("open62541", Log::INF);
```

Comment:
Might it be a good idea to register the LogIt component before registering the _logFromOpen62541_ function with open6?

Not sure where open6 will then call that log function (other threads etc.). LogIt just ignores requests to log to handles it doesn't know, but maybe there is potential for losing open6 log messages.

Reply:
At this stage the logger won't yet be used (so what I propose is safe IMO). It starts to be used only when open62541 server starts. But for the sake of readability (and logical flow) I think you've got the point so I will swap the order there.